src/modules/aws_sdk/index.ts

```diff
@@ -26,6 +26,7 @@ export class AwsSdkModule extends ModuleBase {
   invokeS3: AwsSdkInvoker;
   invokeSecretManager: AwsSdkInvoker;
   invokeSns: AwsSdkInvoker;
+  invokeSts: AwsSdkInvoker;
```

Comment:
Does this work? Why does it say `Sts` instead of `Sqs`?

Reply:
It does. It's for the `INSERT` trigger I already added (to get the account id using `sts.getCallerIdentity` function.